src/pages/markets/static/content/_cryptocurrencies.tsx

```diff
@@ -12,23 +12,23 @@ export type CryptocurrenciesContent = {
 
 export const simple_step_content_cryptocurrencies = [
     {
-        header: <Localize translate_text="Practise" />,
+        header: <Localize translate_text="_t_Practise_t_" />,
```

Comment:
can we please use the `TString` approach here as well? 
just update the data and update the component using these data to loop over it? 
thanks man :pray: